ControlMeasurements/Models/MeasurementCategory.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace ControlMeasurements.Models
+{
+    public class MeasurementCategory
```

Comment:
Tak jak w komentarzu powyżej - IMO nazwa Measurement brzmi lepiej, ale to rzecz kosmetyczna

Reply:
Zmienione :)